src/openrct2/util/util.h

```diff
@@ -56,4 +56,12 @@ uint32 util_rand();
 uint8 *util_zlib_deflate(uint8 *data, size_t data_in_size, size_t *data_out_size);
 uint8 *util_zlib_inflate(uint8 *data, size_t data_in_size, size_t *data_out_size);
 
+#define add_clamp(value, value_to_add, min_cap, max_cap) \
+	if ((max_cap - abs(value_to_add)) < abs(value) && (value < 0) == (value_to_add < 0)) { \
```

Comment:
I did notice that when testing by setting the money value to the lowest negative number possible and then adding "-1" using the add/set money cheats, that it looped back into the positive, but I'm unsure of how to actually address this while keeping this code as clean as it is.. any suggestions?